sky/skylet/log_lib.py

```diff
@@ -184,8 +184,13 @@ def run_with_log(
             daemon_script = os.path.join(
                 os.path.dirname(os.path.abspath(job_lib.__file__)),
                 'subprocess_daemon.py')
+            python_path = subprocess.check_output(
```

Comment:
If this somehow failed, fallback to `python3`? Or maybe it makes sense to crash?

Reply:
We already fallback to python3 in the command. It might be fine to crash if the failure really happens.